### DISCUSSION TITLE
Anyone have Transmission working on Windows 11?

### DISCUSSION THREAD

**Discussion by hl2guide** [transmission/transmission#6377] [Q&A]
I've not been able to get Transmission working correctly on Windows 11.

Is there a trick to get the port open and working properly?

I don't run into this issue with qbittorrent.

I've tried creating Windows Firewall rules but this does not fix it for me 😔

**Comment by NovHak-Linux** [comment]
That may have something to do with the authorised port range for UPNP redirection on your router, with qbittorrent using a port in the authorised range by default

**Comment by Pentaphon** [comment]
I have no issue opening a port using Windows 11. Check your Windows Firewall and router firewall settings. Try not to use the default port as that will someday be randomized for new installs.
